scripts/lotus-daemon.service

```diff
@@ -1,19 +1,17 @@
 [Unit]
 Description=Lotus Daemon
-After=network-online.target
-Requires=network-online.target
+After=network.target
 
 [Service]
-Environment=GOLOG_FILE="/var/log/lotus/daemon.log"
-Environment=GOLOG_LOG_FMT="json"
+User=fc
+Group=fc
```

Comment:
We can either:

- Use a dynamic user.
- Install a systemd-sysusers.

Also, we should call it lotus, not just "fc" (which could mean anything).

Reply:
Sure, this can change it to lotus. There is already "fc" being used elsewhere, so I was continuing to do that, but I have no preference for it and I do think calling it lotus is better. I would not be opposed to systemd-sysuers either.

Reply:
Hm. Where are we using "fc"? I'm just concerned that users will be confused if they later look at their passwd file and find "fc".